test/testmachinery/shoots/logging/seed_logging_stack.go

```diff
@@ -397,9 +401,28 @@ func prepareShootLokiStatefulSet(shootLokiSts, gardenLokiSts *appsv1.StatefulSet
 	shootLokiSts.Spec.Template.Labels = newLabels
 	// Move the shoot Loki in the garden namespace
 	shootLokiSts.Namespace = v1beta1constants.GardenNamespace
+
+	capLokiMemoryAndCPURequests(shootLokiSts)
+
 	return shootLokiSts
 }
 
+func capLokiMemoryAndCPURequests(shootLokiSts *appsv1.StatefulSet) {
```

Comment:
It is not only the Shoot loki, you can the same func for the garden/central loki.

```suggestion
func capLokiMemoryAndCPURequests(lokiSts *appsv1.StatefulSet) {
```